librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -993,6 +1002,11 @@ void QC_ApplicationWindow::slotWindowActivated(QMdiSubWindow* w) {
     activedMdiSubWindow=w;
 
     QC_MDIWindow* m = qobject_cast<QC_MDIWindow*>(w);
+    if (w->windowTitle().contains("Block")) {
```

Comment:
This is highly insecure.
- The title is set with `tr("Block '%1'")`, thus in translated environment a block window title must not contain "Block"
- When the drawing name itself contains "Block" we'll have wrong interpretation too